Can't retrieve mixtool using go get

Hi,
i try to retrieve mixtool, i've got this error : 

```
$ go version
go version go1.15.3 linux/amd64
$ go get -u github.com/monitoring-mixins/mixtool/cmd/mixtool
# github.com/monitoring-mixins/mixtool/cmd/mixtool
golang/src/github.com/monitoring-mixins/mixtool/cmd/mixtool/generate.go:32:22: cannot use cli.StringSliceFlag literal (type cli.StringSliceFlag) as type cli.Flag in slice literal:
        cli.StringSliceFlag does not implement cli.Flag (Apply method has pointer receiver)
golang/src/github.com/monitoring-mixins/mixtool/cmd/mixtool/generate.go:35:3: undefined: cli.BoolTFlag
golang/src/github.com/monitoring-mixins/mixtool/cmd/mixtool/generate.go:44:15: cannot use cli.Command literal (type cli.Command) as type *cli.Command in slice literal
golang/src/github.com/monitoring-mixins/mixtool/cmd/mixtool/generate.go:47:18: cannot use cli.StringFlag literal (type cli.StringFlag) as type cli.Flag in append:
        cli.StringFlag does not implement cli.Flag (Apply method has pointer receiver)
golang/src/github.com/monitoring-mixins/mixtool/cmd/mixtool/generate.go:55:15: cannot use cli.Command literal (type cli.Command) as type *cli.Command in slice literal
golang/src/github.com/monitoring-mixins/mixtool/cmd/mixtool/generate.go:58:18: cannot use cli.StringFlag literal (type cli.StringFlag) as type cli.Flag in append:
        cli.StringFlag does not implement cli.Flag (Apply method has pointer receiver)
golang/src/github.com/monitoring-mixins/mixtool/cmd/mixtool/generate.go:66:15: cannot use cli.Command literal (type cli.Command) as type *cli.Command in slice literal
golang/src/github.com/monitoring-mixins/mixtool/cmd/mixtool/generate.go:69:18: cannot use cli.StringFlag literal (type cli.StringFlag) as type cli.Flag in append:
        cli.StringFlag does not implement cli.Flag (Apply method has pointer receiver)
golang/src/github.com/monitoring-mixins/mixtool/cmd/mixtool/generate.go:77:15: cannot use cli.Command literal (type cli.Command) as type *cli.Command in slice literal
golang/src/github.com/monitoring-mixins/mixtool/cmd/mixtool/generate.go:80:18: cannot use cli.StringFlag literal (type cli.StringFlag) as type cli.Flag in append:
        cli.StringFlag does not implement cli.Flag (Apply method has pointer receiver)
golang/src/github.com/monitoring-mixins/mixtool/cmd/mixtool/generate.go:77:15: too many errors
```

Any idea ? 
what do you think about provides binaries from releases ?

--- a/README.md
+++ b/README.md
@@ -7,8 +7,10 @@ The mixtool is a helper for easily working with [jsonnet](http://jsonnet.org/) m
 ## Install
 
 ```
-go get -u github.com/monitoring-mixins/mixtool/cmd/mixtool
+GO111MODULE=on go get -u github.com/monitoring-mixins/mixtool/cmd/mixtool
 ```
+
+NB, if you see an error of the form `cannot use cli.StringSliceFlag literal (type cli.StringSliceFlag) as type cli.Flag in slice literal`, you didn't set `GO111MODULE=on` before `go get` (see #27).
 
 ## Usage
 
@@ -55,7 +57,7 @@ COMMANDS:
 
 OPTIONS:
    --help, -h  show help
-   
+
 ```
 
 ### New
@@ -75,7 +77,7 @@ COMMANDS:
 
 OPTIONS:
    --help, -h  show help
-   
+
 ```
 
 #### New Examples
@@ -103,7 +105,7 @@ OPTIONS:
    --grafana                Lint Grafana dashboards against Grafana's schema
    --prometheus             Lint Prometheus alerts and rules and their given expressions
    --jpath value, -J value  Add folders to be used as vendor folders
-   
+
 ```
 
 #### Lint Examples

